proof-producer/libs/preset/include/nil/proof-generator/preset/bytecode.hpp

```diff
@@ -0,0 +1,67 @@
+#ifndef PROOF_GENERATOR_LIBS_PRESET_BYTECODE_HPP_
+#define PROOF_GENERATOR_LIBS_PRESET_BYTECODE_HPP_
+
+#include <boost/log/core.hpp>
+#include <boost/log/expressions.hpp>
+#include <boost/log/trivial.hpp>
+#include <nil/blueprint/blueprint/plonk/assignment.hpp>
+#include <nil/blueprint/zkevm/bytecode.hpp>
+#include <optional>
+#include <string>
+
+namespace nil {
+    namespace proof_generator {
+        template<typename BlueprintFieldType>
+        std::optional<std::string> initialize_bytecode_circuit(
+                std::optional<nil::crypto3::zk::snark::plonk_constraint_system<BlueprintFieldType>>& bytecode_circuit,
+                std::optional<nil::crypto3::zk::snark::plonk_table<BlueprintFieldType, nil::crypto3::zk::snark::plonk_column<BlueprintFieldType>>>& bytecode_table) {
+
+            using ArithmetizationType =
+                    nil::crypto3::zk::snark::plonk_constraint_system<BlueprintFieldType>;
+
+            using ComponentType = nil::blueprint::components::zkevm_bytecode<ArithmetizationType, BlueprintFieldType>;
+
+            // initialize assignment table
+            bytecode_circuit.emplace();
+            bytecode_table.emplace(65,  // witness
```

Comment:
Maybe this function needs to accept a table_description argument, which will contain these constants? And if you want you can have a default value for that argument temporarily.